pkg/simple/client/auditing/elasticsearch/options.go

```diff
@@ -52,7 +59,12 @@ func (s *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
 	fs.BoolVar(&s.Enable, "auditing-enabled", c.Enable, "Enable auditing component or not. ")
 
 	fs.StringVar(&s.WebhookUrl, "auditing-webhook-url", c.WebhookUrl, "Auditing wehook url")
-
+	fs.IntVar(&s.GoroutinesNum, "auditing-goroutines-num", c.GoroutinesNum,
+		"The number of goroutines which send auditing events to webhook.")
+	fs.IntVar(&s.MaxBatchSize, "auditing-batch-max-size", c.MaxBatchSize,
+		"The max size of the auditing event in a batch.")
+	fs.DurationVar(&s.MaxBatchWait, "auditing-batch-max-wait", c.MaxBatchWait,
+		"MaxBatchWait indicates the maximum interval between two batches.")
```

Comment:
MaxBatchWait indicates the maximum interval between two batches. => The batch interval of auditing events.